xml/System.Diagnostics.Tracing/IncrementingEventCounter.xml

```diff
@@ -79,8 +79,8 @@
         <Parameter Name="increment" Type="System.Double" />
       </Parameters>
       <Docs>
-        <param name="increment">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="increment">The value to increment by.</param>
+        <summary>Writes 'value' to the stream of values tracked by the counter.  This updates the sum and other statistics that will  be logged on the next timer interval.</summary>
```

Comment:
```suggestion
        <summary>Adds 'increment' to a running total that will be published by the counter at the next timer interval</summary>
```

Reply:
Thanks! Since 'increment' is a parameter, it should be referenced using paramref, like this:

```suggestion
        <summary>Adds <paramref name="increment" /> to a running total that will be published by the counter at the next timer interval.</summary>
```